strongbox-web-core/src/test/java/org/carlspring/strongbox/security/CustomAccessDeniedHandlerTest.java

```diff
@@ -62,7 +62,7 @@ public void customAccessDeniedHandlerShouldRespondAsExpected()
     @WithMockUser(username = "unauthorizedUser")
     public void unauthorizedUserShouldReceiveExpectedUnauthorizedResponse()
     {
-        given().contentType("application/json")
+        mockMvc.contentType("application/json")
```

Comment:
Could you use `MediaType.APPLICATION_JSON_VALUE` instead of the String constant? Thanks!